src/lilURL.php

```diff
@@ -61,13 +69,46 @@ public function handleRedirect($id)
         if ($id != '' && $id != basename($_SERVER['PHP_SELF']) && $id != '?login') {
             $location = $this->getURL($id);
             if ($location != false) {
+                $this->trackHit($id);
```

Comment:
Your concern does not escape me. This is indeed quick and dirty. Although, I have more trust in a Google service for being responsive.

The async route won't help as the request will wait for completion as soon as the object is destructed (at the end of the method). Batching would be a better implementation, but has it's own complications including establishing a cron process and creating the storage mechanism for the hit backlog.

My goal, to at least get something started, was to do something that didn't require new storage or an additional, external process.